Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -251,6 +252,7 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
         let cell = tableView.dequeueReusableCellWithIdentifier(CellIdentifier) as CustomCell
         cell.titleText.text = tab.title
         cell.background.image = tab.screenshot(size: CGSize(width: tableView.frame.width, height: TabTrayControllerUX.CellHeight))
+        cell.accessibilityLabel = tab.url?.absoluteString
```

Comment:
I think this is wrong for a couple of reasons:
 1. I think people who need accessibility queues will want the same information as people without them. That is, the cell text is the tab title, so wouldn't it make sense to read them the title instead of something else?
 1. But why would we need to add this in the first place? We should have tests verify cells based on what the user sees (the table title) rather than some invisible underlying data.